doc/api/events.md

```diff
@@ -694,11 +694,15 @@ added: v11.13.0
 * `name` {string}
 * Returns: {Promise}
 
-Creates a `Promise` that is resolved when the `EventEmitter` emits the given
+Creates a `Promise` that is fulfill when the `EventEmitter` emits the given
```

Comment:
Why this change? I think 'resolved' is much more in line with `Promise` terminology, especially with its `.resolve()` method.

Reply:
Fwiw, if this stays, there’s a typo:

```suggestion
Creates a `Promise` that is fulfilled when the `EventEmitter` emits the given
```

(I’m okay with this change since emitting the event will never result in a rejection, so fulfill is clearer to me.)

Reply:
a resolved promise might fulfill or reject

Reply:
Fulfilled is correct here. 

OP based this on a PR I opened a while back and with my blessing :]

@mscdex 

> Why this change? I think 'resolved' is much more in line with Promise terminology, especially with its resolve() method.

`resolve` means "to assume the state of another value, you can `resolve` with another promise, a value, etc. In a gist:

A promise starts off as *pending* and transitions into one of two states:
 - *fulfilled* which is successful completion. This models a function returning a value.
 - *rejected* which is unsuccessful completion. This models a function throwing an error.

A promise is *settled* if it transitioned from pending to fulfilled or rejected.

A promise is *resolve*d if its value is decided, that is it is fulfilled, rejected or it follows another promise.

Here is an example of a promise that is resolved but not fulfilled for example:

```
const p = new Promise(resolve => resolve(fetch(...));
```

`p` is resolved to the inner promise passed to the fetch. The reason `resolve` is so common is because the promise constructor _does_ resolve, for example the following creates a rejected promise:

```
const p = new Promise(resolve => resolve(Promise.reject(new Error(''))));
```

Now normally I don't care very much about this terminology - but given that we _are_ the platform we should strive to be precise when we can :]